### PR TITLE
Deploy with Travis instead of Github actions

### PR DESCRIPTION
**Pull request by hannalaakso**
### Context
As we no longer use Github actions, change the app to deploy to PaaS using Travis instead.

### Changes proposed in this pull request
- Remove Github actions workflow
- Add a Travis config to
 	- Set encrypted CloudFoundry credentials as env variables
 	- Install CloudFoundry CLI tools
- Add an executable deploy script which uses CloudFoundry zero-downtime push

The change was tested to work by temporarily allowing deploys from any branch into a test site; see [test build](https://travis-ci.com/alphagov/prototype-kit-training/builds/138594239), under Deploying application (afterwards reverted to limit deploys to happen from master branch and to deploy the live site).

Fixes https://github.com/alphagov/prototype-kit-training/issues/9

### CODE REVIEW

**Comment by 36degrees** [comment]
@nickcolley as I worked on this with @hannalaakso, it'd be great if you could review this when you've got some time.

You can see a [successful build](https://travis-ci.com/alphagov/prototype-kit-training/builds/138594239#L495) in Travis to a test application before we re-enabled the branch restrictions.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
 cache: bundler
 script: bundle exec middleman build
+
+env:
+  global:
+  # CloudFoundry credentials for deployment
+  - CF_API="https://api.cloud.service.gov.uk"
+  - CF_ORG="govuk-design-system"
+  - CF_SPACE="prototype-kit-training"
+  - CF_USERNAME="prototype-kit-docs-deploy-production@digital.cabinet-office.gov.uk"
```
Comment (nickcolley) [inline .travis.yml]: Did you setup a new email or did this always exist? is this in the credentials repo?

Comment (36degrees) [reply .travis.yml]: https://github.com/alphagov/design-system-team-credentials/tree/master/paas/prototype-kit-docs

Comment (hannalaakso) [reply .travis.yml]: It's been added to the credentials repo as part of this work 👍

**Pull request merged**